### PR TITLE
embeddings: nomic embed vision

### PR DESCRIPTION
**Pull request by zanussbaum**
Thank you for contributing to LangChain!

**Description:** Adds Langchain support for Nomic Embed Vision
**Twitter handle:** nomic_ai,zach_nussbaum

- [x] **Add tests and docs**: If you're adding a new integration, please include
  1. a test for the integration, preferably unit tests that do not rely on network access,
  2. an example notebook showing its use. It lives in `docs/docs/integrations` directory.

- [ ] **Lint and test**: Run `make format`, `make lint` and `make test` from the root of the package(s) you've modified. See contribution guidelines for more: https://python.langchain.com/docs/contributing/

Additional guidelines:
- Make sure optional dependencies are imported within a function.
- Please do not add dependencies to pyproject.toml files (even optional ones) unless they are required for unit tests.
- Most PRs should not touch more than one package.
- Changes should be backwards compatible.
- If you are adding something to community, do not re-import it in langchain.

If no one reviews your PR within a few days, please @-mention one of baskaryan, efriis, eyurtsev, ccurme, vbarda, hwchase17.

### CODE REVIEW

File: libs/partners/nomic/langchain_nomic/embeddings.py [hunk 1]
```diff
@@ -121,3 +124,27 @@ def embed_query(self, text: str) -> List[float]:
             texts=[text],
             task_type="search_query",
         )[0]
+
+
+class NomicMultimodalEmbeddings(NomicEmbeddings):
```
Comment (baskaryan) [inline libs/partners/nomic/langchain_nomic/embeddings.py:129]: might it be a simpler / more intuitive devx if NomicEmbeddings just implemented the multimodal functionality?

Comment (zanussbaum) [reply libs/partners/nomic/langchain_nomic/embeddings.py]: updated!

**Review by rlancemartin — COMMENTED** [review]
In the notebook, clear cell outputs to clean. Showcasing on more practical example w/ evaluation would be great, as noted from Slack.

[https://langchain-ai.github.io/langchain-benchmarks/notebooks/retrieval/multi_modal_benc[…]ing/multi_modal_eval_baseline.html?ref=blog.langchain.dev](https://langchain-ai.github.io/langchain-benchmarks/notebooks/retrieval/multi_modal_benchmarking/multi_modal_eval_baseline.html?ref=blog.langchain.dev)

**Review by rlancemartin — APPROVED** [review]
LGTM. Only nit: in README, show full usage for OpenCLIP since we pulled that out.

preserves the ability for people to know how to use it.  

```
langchain_experimental.open_clip import OpenCLIPEmbeddings
embedding_function=OpenCLIPEmbeddings(
        model_name="ViT-H-14", checkpoint="laion2b_s32b_b79k")
```

**Pull request merged**